src/main/java/seedu/address/logic/parser/AddMeetingCommandParser.java

```diff
@@ -17,13 +21,21 @@ public class AddMeetingCommandParser implements Parser<AddMeetingCommand> {
     @Override
     public AddMeetingCommand parse(String args) throws ParseException {
         ArgumentMultimap argMultimap =
-                ArgumentTokenizer.tokenize(args, PREFIX_NAME, PREFIX_DATE, PREFIX_TIME);
+                ArgumentTokenizer.tokenize(args, PREFIX_INDEX, PREFIX_DATE, PREFIX_TIME);
+
+        if (!arePrefixesPresent(argMultimap, PREFIX_INDEX, PREFIX_DATE, PREFIX_TIME)
+                || !argMultimap.getPreamble().isEmpty()) {
```

Comment:
Instead of checking whether `getPreamble` is empty, maybe can just standardise it by using `requireNonNull` at the start which is similar to other files?